ci-operator/config/openshift/pagerduty-short-circuiter/openshift-pagerduty-short-circuiter-main.yaml

```diff
@@ -12,8 +12,24 @@ resources:
       cpu: 100m
       memory: 200Mi
 tests:
+- as: build
```

Comment:
not sure what is happenning, but you made those changes in a config from another repo. Maybe a bad rebase?